scrapy/utils/gz.py

```diff
@@ -51,8 +51,12 @@ def gunzip(data):
     return output
 
 _is_gzipped_re = re.compile(br'^application/(x-)?gzip\b', re.I)
+_is_octetstream_re = re.compile(br'^(application|binary)/octet-stream\b', re.I)
```

Comment:
@redapple The PR looks fine :+1:. Could you please clean up and merge it?

2016-07-09 4:25 GMT+05:00 Joakim Uddholm <notifications@github.com>:

> In scrapy/utils/gz.py
> <https://github.com/scrapy/scrapy/pull/2065#discussion_r70154620>:
>
> > @@ -51,8 +51,12 @@ def gunzip(data):
> >      return output
> >
> >  _is_gzipped_re = re.compile(br'^application/(x-)?gzip\b', re.I)
> > +_is_octetstream_re = re.compile(br'^(application|binary)/octet-stream\b', re.I)
>
> Doing is not None explicitly makes it a bool though. If we drop the is
> not None then we're going to return None or True instead of True or
> False. I doubt it makes a difference though.
>
> return _is_gzipped(ctype) or
>             (_is_octetstream(ctype) and
>              cenc in (b'gzip', b'x-gzip')))
>
> If we want to be really pedantic we could probably also drop the
> parantheses too.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/scrapy/scrapy/pull/2065/files/778f1cf84cfc2047fdd43b45f908bf0078a160b6#r70154620>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGldbtIqFZwferWdBSBy2TG31Tv_0zEks5qTtxngaJpZM4I5xtb>
> .
>